datafusion/physical-plan/src/aggregates/group_values/group_value_row.rs

```diff
@@ -60,11 +61,13 @@ pub trait ArrayRowEq: Send + Sync {
     fn take_n(&mut self, n: usize) -> ArrayRef;
 }
 
+/// An implementation of [`ArrayRowEq`] for primitive types.
```

Comment:
was it renamed in https://github.com/apache/datafusion/pull/12619?

Reply:
Yes, good call -- will fix